db/migrate/20190715164535_add_instance_actor.rb

```diff
@@ -0,0 +1,9 @@
+class AddInstanceActor < ActiveRecord::Migration[5.2]
+  def up
+    Account.create!(id: -99, actor_type: 'Application', locked: true, username: Rails.configuration.x.local_domain)
```

Comment:
You need to add this to `seed.rb` as well for new installations.

Reply:
Oops. I will do that!